e2e/wrappers/builders/react19/src/pages/ComponentView.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { Suspense, useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { findComponentByPath } from '../utils/componentFinder';
+
+export default function ComponentView() {
+  const { examplePath } = useParams();
+  const [Component, setComponent] = useState(null);
+
+  useEffect(() => {
+    const componentData = findComponentByPath(examplePath);
+
+    componentData.component()
+      .then(module => {
+        setComponent(() => module.default);
+        setLoading(false);
```

Comment:
`setLoading` looks undefined here, it should throw an error if I'm not missing anything

Before even fixing this line, I wonder do we see this error on CI, and if not, why not?